zerver/lib/bugdown/__init__.py

```diff
@@ -754,11 +754,14 @@ def handleMatch(self, match):
 def make_emoji(codepoint, display_string):
     # type: (Text, Text) -> Element
     src = '/static/generated/emoji/images/emoji/unicode/%s.png' % (codepoint,)
+    # Replace underscore in emoji's title with space
+    title = display_string[1:-1].replace("_", " ")
+
     elt = markdown.util.etree.Element('img')
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
```

Comment:
Hmm, well if we just set `alt`, then mousing over the item.  So maybe we do need both?  

I guess these are potentially worth looking at:
https://tink.uk/accessible-emoji/
http://adrianroselli.com/2016/12/accessible-emoji-tweaked.html
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/accessible-emoji.md

Either way, it looks like the preference would be to have both be the value we're setting as title, right?